### PR TITLE
Feature/sign up screens

### PR DESCRIPTION
**Pull request by thiagoandf**

### CODE REVIEW

File: app/containers/SignupExperience/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { compose } from 'redux';
+import SignupExperience from '../../ui/pages/SignupExperience';
+import { addHistoryAndPush } from '../../state/registration/actions';
+
+const mapDispatchToProps = dispatch => ({
+  addTheFuckingHistory: (weeklyPlaytime, collectionSize) =>
```
Comment (caiorulli-quintoandar) [inline app/containers/SignupExperience/index.js:8]: NICE YO

File: app/ui/pages/SignupExperience.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        </AppBar>
+        <div className={classes.titleWrapper}>
+          <Typography
+            variant="title"
+            style={{ textAlign: 'center', paddingBottom: '30px' }}
+          >
+            Casual ou Hardcore?
+          </Typography>
+          <Typography variant="subheading" style={{ textAlign: 'center' }}>
+            Agora arraste os seletores abaixo selecionando quanto você acha que
```
Comment (caiorulli-quintoandar) [inline app/ui/pages/SignupExperience.js:97]: Move to messages :)

File: package.json [hunk 3]
```diff
@@ -138,6 +138,7 @@
   "dependencies": {
     "@material-ui/core": "^1.3.0",
     "@material-ui/icons": "^1.1.0",
+    "@material-ui/lab": "^3.0.0-alpha.19",
```
Comment (caiorulli-quintoandar) [inline package.json:141]: Alpha, perfect!!!

**Review by caiorulli-quintoandar — APPROVED** [review]

File: app/ui/pages/SignupExperience.js (hunk 2 above)
Comment (thiagoandf) [reply app/ui/pages/SignupExperience.js:97]: No

**Pull request merged**